x-pack/plugins/apm/e2e/cypress/integration/rum_dashboard.feature

```diff
@@ -5,3 +5,19 @@ Feature: RUM Dashboard
     When the user inspects the real user monitoring tab
     Then should redirect to rum dashboard
       And should have correct client metrics
+
+  Scenario: Page load distribution
+    Given a user browses the APM UI application for RUM Data
+    When the user inspects the real user monitoring tab
+    Then should redirect to rum dashboard
```

Comment:
If you take a look, these 3 steps all-together have been tested in the above scenario. What do you think about this scenario, rephrased:

```gherkin
Scenario: Page load distribution
Given a valid APM user
When the APM user inspects the RUM dashboard
Then the "page load" chart includes "percentile"
   And the "page load" chart includes "tooltip"
   And the "page load" chart includes "chart legend"
```

Using quotes in a step generates a generic method in the implementation code, where the quoted string is replaced by a variable:

```javascript
Then(`the "{string}" chart includes {string}`, (chartName, element) => {
  // cypress code using "chartName" and "element" variables
  // i.e. locate the "chartName" chart and check for the "element"
})
```